.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+### Summary
+
+Add a short summary of the entire issue in 1-3 sentences in the first paragraph.
+
+Add a more thorough description of the issue in any length in the second paragraph (when required)
+
+### Analysis
+
+- If you're reporting an error, provide a link that shows where to observe the problem.
```

Comment:
Given that we ask for a link we may not need to explicitly call out the language. It may be worth being explicit given the funnel here. May also want to leverage the checklisting feature here.

Reply:
Given that the plan and organization of the docs are allowing for different translations it is likely that we will have feedback based on specific languages. The comment is that we can likely derive the language from the url, but given that we are currently looking at what are effectively iso languages codes this can not be immediately clear. It may be beneficial to have the language called out explicitly.

Reply:
Ohh.. sorry, I missed context earlier! I've removed the initially planned multi-language feature for now, in order to focus our limited resources on docs quality and volume first.